lib/reportportal.rb

```diff
@@ -101,10 +101,14 @@ def send_log(status, message, time)
     def send_file(status, path, label = nil, time = now, mime_type = 'image/png')
       url = "#{Settings.instance.project_url}/log"
       unless File.file?(path)
+        if mime_type =~ /;base64$/
+          mime_type = mime_type[0..-8]
+          path = Base64.decode64(path)
```

Comment:
`require 'base64'` is missed

Reply:
we didn't have `require 'base64'` before and it used to work